fpga_interchange/fasm_generators/generic.py

```diff
@@ -348,13 +348,16 @@ def fill_features(self):
         """
         pass
 
-    def output_fasm(self, fasm_file):
+    def output_fasm(self, elapsed_time, fasm_file):
         """
         Outputs FASM features that were filled during the fill_features call
         """
 
         with open(fasm_file, "w") as f:
-            print(self.get_origin_line(), file=f)
+            print(
+                "{} - runtime (s): {:.3f}".format(self.get_origin_line(),
```

Comment:
worth noting that including the runtime makes the output FASM non-deterministic, e.g. if you want to compare hashes

Reply:
Hmm right, probably it might be best leaving this out for the tool that call the fasm generator to determine how much time is spent, or in alternative produce a log file with some metadata